content/modules/mobile-travel-tutorial/partials/installation/manual.adoc

```diff
@@ -71,32 +71,65 @@ git clone -b master --depth 1 https://github.com/couchbaselabs/mobile-travel-sam
 
 == Couchbase Server
 
+In this lesson, you will install and launch Couchbase Server.
 
-In this lesson, you will install and launch version 7.0.0 of Couchbase Server.
-If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with the 7.0.0 version.
+If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with a certain version.
 Note that the actual UI may vary slightly depending on the version of Couchbase server that you have installed.
 
-* https://www.couchbase.com/downloads#couchbase-server[Download and install] v7.0.x of Couchbase Server.
+=== M1 Users
```

Comment:
Maybe I am missing something here. Other than the version of CBS that is deployed, is there anything that is different between the instructions for M1 and other platforms as far as CBS setup goes? 

If not, I would suggest highlighting that M1 needs min Couchbase Server version of 7.1.1

Reply:
no, there's no other difference. I will update

Reply:
Should we keep both flavours? Or I can drop the M1 one and just keep the default - or the other way around (screenshots wise)